### ISSUE TITLE
Add the xeno half-bright to ghosts and Aghosts in the toggle lighting setting

### ISSUE THREAD

**Issue by XxSWAG-MASTER-OVERLORDxX** [RMC-14/RMC-14#8095]
## Description
Xenos now have access to a "Half-bright" that lets them see much better than how a Full-bright in most lighting conditions, it would be great if both ghosts and admins had access to it so their eyes were not bleeding out of their head from full-bright.

**Comment by snebl** [comment]
should be closed now that its added.
